src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Float.cs

```diff
@@ -99,21 +100,17 @@ public static bool TryFormat(float value, Span<byte> destination, out int bytesW
                 utf16Text = value.ToString(new string(formatText), CultureInfo.InvariantCulture);
             }
 
-            // Copy the value to the destination, if it's large enough.
+            bytesWritten = 0;
```

Comment:
Move this into the catch and the return true into the try?

Reply:
Fixed.